### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by awoimbee**
Hi,
I found there were many typos, this fixes most of them !

Note:
The best way to view the diff in `lasso/dimred/svd/html_str_eles.py` is `git diff --word-diff-regex=. origin/main -- lasso/dimred/svd/html_str_eles.py` (the only change is `menue` -> `menu`).

### CODE REVIEW

File: lasso/dimred/svd/html_str_eles.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
...ion-color);text-decoration:underline;text-decoration-thickness:3px;text-underline-offset:5px}#resetPlot{position:absolute;margin-left:30px;color:var(--plot-option-color);font-size:35px}#sizeDrag{cursor:col-resize;width:2%}#dragLineDiv{width:0;border:var(--content-border);margin-left:45%;margin-right:49%}.imgDiv{border-radius:5px;box-shadow:var(--content-shadow);border:var(--content-border);display:flex;justify-content:center;align-items:center;background-color:var(--content-background)}.alignedIMG{width:99%;height:auto}.imgDiv :is(p,h2){margin-top:0;padding-left:15px;display:inline-block;vertical-align:middle}.noImgDescr{position:relative;display:none}.traceContainerClass{float:left;padding-bottom:5px}.traceContainerClass :is(p,canvas){float:left}.traceContainerClass p{font-size:16px;padding-left:5px;padding-right:50px;color:var(--menu-option-text-color);margin-top:15px;opacity:.7}.traceContainerClass p:hover{opacity:.9}.traceContainerClass canvas{border-radius:24px;margin-bottom:15px;margin-top:5px}.traceContainerClass canvas:hover{box-shadow:0 0 3px 2px #12678f;transition:.4s}.colorwheel{border-radius:128px!important;border-color:#f1f1f1;position:relative;z-index:10;top:0;left:0}
```
Comment (codie3611) [inline lasso/dimred/svd/html_str_eles.py:7]: Any idea what happened here?

Comment (codie3611) [reply lasso/dimred/svd/html_str_eles.py:7]: Ah I see it is the menu issue again. Did you verify it works fine?

Comment (awoimbee) [reply lasso/dimred/svd/html_str_eles.py:7]: Ah, I edited my PR description after your review, you're too fast ! So:

> The best way to view the diff in `lasso/dimred/svd/html_str_eles.py` is `git diff --word-diff-regex=. origin/main -- lasso/dimred/svd/html_str_eles.py` (the only change is `menue` -> `menu`).

**Review by codie3611 — APPROVED** [review]
Thanks for the work

**Pull request merged**